src/main/mixpanel.ts

```diff
@@ -0,0 +1,45 @@
+import { Mixpanel } from "mixpanel";
+
+export interface INineChroniclesMixpanel {
```

Comment:
Do we have a reason to declare interface rather than using class type directly?

Reply:
This code is intended to make writing test code a little easier. I thought this would make it easier to isolate dependencies.

Reply:
You don't need to worry about that case, since TypeScript uses structual typing on classes just like interfaces.

https://www.typescriptlang.org/docs/handbook/2/classes.html#relationships-between-classes

Reply:
That is awesome! But in this case, I think it is easy to get help from the IDE which Mocking class has the same interface as the corresponding class.

Reply:
You can check [__tests__/mock/MockNineChroniclesMixpanel.ts](https://github.com/planetarium/9c-launcher/pull/875/commits/4cad1598e42db4ab6f9df14a04b5bfef23e07d13) too.